lib/buffer.js

```diff
@@ -1024,6 +1024,13 @@ function checkOffset(offset, ext, length) {
     throw new errors.RangeError('ERR_INDEX_OUT_OF_RANGE');
 }
 
+function checkByteLength(byteLength) {
+  if (byteLength < 1 || byteLength > 6)
+    throw new errors.RangeError('ERR_OUT_OF_RANGE',
+                                'byteLength',
+                                'between 1 and 6');
```

Comment:
I think the existing `ERR_OUT_OF_RANGE` uses the syntax `>= 1 and <= 6`, so I would be consistent with others here.

Reply:
Changed, thanks!